src/main/java/com/epam/aidial/cfg/service/transfer/importer/KeyImporter.java

```diff
@@ -71,9 +71,11 @@ public Collection<ImportComponent<Key>> importAdminKeys(Map<String, Key> keys,
     private ImportComponent<Key> processKey(Key key, ConflictResolutionPolicy resolutionPolicy) {
         Optional<Key> existingKey = keyService.tryGetKey(key.getName());
         if (existingKey.isPresent()) {
+            key.setDisplayName(existingKey.get().getDisplayName());
             ImportAction importAction = handleExisting(key, resolutionPolicy);
             return new ImportComponent<>(importAction, existingKey.get(), key);
         } else {
+            key.setDisplayName(key.getKey());
```

Comment:
`key.getKey()` is secretValue. it's bad idea to use it as displayName

Reply:
@siarhei-fedziukovich, @KirylKurnosenka, how to fill the DisplayName attribute for the key during import?

Reply:
can we use key.setDisplayName(key.getName()) ? how did we generate key.name here?